app/models/invenio_rdm_record_converter.rb

```diff
@@ -173,9 +173,10 @@ def invenio_metadata
         "creators": @generic_file.creator.map{ |creator| build_creator_contributor_json(creator) },
         "title": @generic_file.title.first,
         "additional_titles": format_additional("title", "alternative-title", @generic_file.title.drop(1)),
-        "description": @generic_file.description.first,
-        "additional_descriptions": format_additional("description", "other", @generic_file.description.drop(1)) + format_additional("description", "acknowledgements", @generic_file.acknowledgments)\
-          + format_additional("description", "abstract", @generic_file.abstract),
+        "description": @generic_file.description.join("\n\n").force_encoding("UTF-8"),
+        "additional_descriptions":
+          format_additional("description", "acknowledgements", @generic_file.acknowledgments) +
```

Comment:
Oh, I see - you were considering the readability of the diff AND the code. I'll keep this in mind if we ever get around to implementing a linter for the website.